client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx

```diff
@@ -56,42 +47,13 @@ const InvestigationForm: React.FC = (): JSX.Element => {
         [exposureDate, symptomsStartDate, hasSymptoms, endInvestigationDate, setSymptomsStartDate, setExposureDate, setHasSymptoms, setEndInvestigationDate]
     );
 
-    const { confirmFinishInvestigation, areThereContacts, setAreThereContacts } = useInvestigationForm();
+    const { areThereContacts, setAreThereContacts } = useInvestigationForm();
     const {
         currentTab,
-        setNextTab
+        setNextTab,
+        lastTabDisplayedId
     } = useTabManagement();
 
-    const lastTabDisplayedId = areThereContacts ? LAST_TAB_ID : LAST_TAB_ID - 1;
-
-    const isInvestigationValid = () => {
-        return !(formsValidations.slice(0, lastTabDisplayedId + 1).some((formValidation) => !formValidation));
-    }
-
-    const handleNextPageClick = () => {
```

Comment:
Why did you delete it?

Reply:
we moved this functionality to useTabManagment in order to call change tab function only after the tab saved completly